### ISSUE TITLE
Black water backing up in toilet bowl

### ISSUE THREAD

**Issue by smr547** [CanberraOceanRacingClub/namadgi3#628]
There have been recent reports of black water reentering the toilet bowl after a clear flushing.

@pauljones17 reports:

>However, Peter and I had also noted a problem with all bowls, that being after pumping the bowl dry, bowls back filled with liquid of various shades.  I surmised it might be a syphoning issue as fluid drained back down. 
Peter thought perhaps it was the joker valve, which should prevent the drain back.  
Either way, whatever makes its way into the bowl presents a serious problem in a seaway because the liquid slops onto the floor (as Michael reported) and is then tramped through the yacht.  Definitely a matter for urgent pre-deployment resolution.

@pjwain has provided an excellent summary of what is happening, Peter writes:

>When the holding tank is full, continued pumping will send black water into the breather. The first sign of this is that an odour can be detected on deck as black water exits from the breather opening. Pumping becomes more and more difficult as the pressure increases, and it is likely to cause a blockage at the breather exit, which has a very fine stainless steel gauze.

>Once the breather is blocked, further any pumping will pressurize the holding tank, causing the sides to bulge out, and is likely to overcome the joker valve, allowing black water to return to the toilet  bowl.

>At that point, any attempt to empty the holding tank is likely to fail. On opening the drain valve, enough black water will be dumped to relieve the pressure, but after that the breather blockage will cause sufficient suction to prevent any further egress of black water from the tank.

>To empty the tank at sea in that situation, the holding tank deck fitting can be opened to allow air into the tank to overcome the suction. Do not open the deck fitting if the holding tank is under pressure! Make sure the drain is open first.

>Even emptying the tank by pumping out can fail because there is not enough suction to overcome the blocked breather. If that happens I'm told that you can empty the tank by opening the drain valve to allow seawater to enter the tank from below, emptying the black water from the tank, but leaving it full of seawatrer, but I haven't tried that myself.

>**The breather blockage can be cleared by aiming high water pressure towards the breather gauze from outside.**

Peter's last dot point is fundamental to solving the problem and preventing it from occurring again.

**Comment by smr547** [comment]
@delcosta (Sam Delorenzo) has made an excellent contribution to the whole issue of the toilets and black water management. Below is his description of the system and its operation supported by a number of  [diagrams](https://github.com/CanberraOceanRacingClub/namadgi3/files/10188654/N3.Black.Water.System.Maintenance.pdf)

>Dear All

There is certainly a need to ensure consistency on how to manage the Black Water system on Namadgi. I have studied our system and experienced (and have cleared) the blockages (our foe).

I offer the attached and below as a point of clarification and maybe assist understanding of a properly working system (our friend) I attach a [series of sketches](https://github.com/CanberraOceanRacingClub/namadgi3/files/10188654/N3.Black.Water.System.Maintenance.pdf) of how I understand the configuration on N3.

The system provides many opportunities for failure, which include.

1. Foreign object tank blockages (wet wipes) – Unfortunately, our tanks will funnel these to outlets (see diagram 5)
2. Calcification - see diagram 2 showing where wastewater sits under normal circumstances and build up
3. Breather line blockage - see diagrams 3 & 5 showing dual breather overflow operation
4. Sea cock blockage at the hull.
5. Failure to verify that tanks have emptied at sea (the golden streak is a beautiful sight)
6. Incorrect/inadequate pumping sequence (there is only one way)
7. Failed seals
8. Black tanks over filled (can block the breather)
9. Combination of any of the above

Note: the best dye/tracing indicator is a litre of milk through the system for observation.

Diagram 7 is a draft of how I think we can maintain a reliable system (when it is clear and unblocked).

I would welcome any comments from experiences.

I am preparing a presentation Live on the boat either during the working bee this month, or suitable date before South Pacific departure, and I will be asking all skippers and interested members to attend so we can all be constantly confident that the system is being maintained as best we can.

It’s quite a simple process of preventative maintenance that must be understood and carried out by all Skippers.

 

 

Regards,

Sam DeLorenzo

**Comment by smr547** [comment]
Firstly, thank you Sam for the brilliant diagrams and descriptions of our black water system. It is a complex system and we have three of them on board.

Now to the immediate problem...

The current problem with the toilet(s) on Namadgi relates to a blocked breather tube. A blocked breather tube leads to a diabolical situation known as a pressure lock

1. Namadgi has NO reliable waste water tank monitoring system, consequently we regularly over-fill the black water tanks,  with a full tank...
2. Continued pumping of waste into the black water tank forces all air out of the system via the breather tube, which then fills with effluent
3. Effluent will spay out of the breather tube sometime attracting the attention (and comments) of nearby swimmers or neighbours enjoying a BBQ  -- a brown stain will run down the topsides
4. This effluent dries at the end of the breather creating a solid plug that will NOT allow air back into the system.
5. Water (and shit) are incompressible
6. Further pumping of effluent causes the plastic black water tank to expand (and actually stretch) placing the vile contents under high pressure
7. Shit under pressure is a dangerous thing
8. Over a short time, this pressure will overcome the joker valve force black water into the bowl -- this is the first visible sign of a major problem -- **Stop using those heads!!**
9. The pressure of the black water at the top of the pump out tube is likely to well exceed the air pressure at the deck
10. Consequently, removal of the deck pump-out cap will cause a geyser of shit to erupt into the air!!  Seriously! **Don't be tempted to open that cap.**

So how to relieve this pressure lock?

1. Opening the black water sea cock may relieve some of the pressure, but
2. It's not unusual for effluent to settle just above the sea cock and for a blockage of it's own, however, more commonly,
3. **a little effluent** will escape to sea, relieving the positive pressure in the top part of the tank and causing the walls of the tank to bow inwards.
4. However, the upper part of the system is still closed, with no air vent to the atmosphere, so the black tank **will not empty to the sea** despite the valve being open
5. Opening the pump out deck fitting at this stage is less dangerous than before but is **a complete waste of time**
6. Sam's diagram shows the horrible reality, the pump out tube loops from the deck fitting to the bottom of the tank...
7. Before air can get into the tank, the effluent in the tube (and there is a lot of it) must enter the black water tank -- but there is no room for that effluent
8. The situation is a perfect pressure lock from which there is no easy exit

Is there a solution? It depends where you are. At sea or away from a marina (where no pump out facility is available)

1. With black water seacock open...
2. Remove the cover to gain access to the black water tank
3. Detach the breather tube from the top of the tank (a Phillips head screwdriver is needed to ease the jubilee clamp)
4. Don't be distressed by the inrush of air as the tanks suddenly expands and the column of shit is dumped to the sea
5. at the earliest opportunity, use a pressure hose to clear the breather line from the outside
6. reattach the breather tube once it has been cleared

Good luck and please let us know how you get on.

regards
Steven

**Comment by smr547** [comment]
Hi Michael,

That makes sense. The pressurised black water tank is forcing effluent back
into the toilet bowl through the joker valve (which is a very inefficient
one-way valve). Once sufficient ugly water has returned to the bowl,
pressure will reduce in the black water tank and the flow will stop.
Bucketing the effluent out is a great idea.... gone but not forgotten!!

Could you please flush fresh sea water into the deck pump out fitting --
two or three buckets please. This will clean out effluent from the pump out
hose loop. All of these pipes are due to be pulled out over the next two
weeks. Have I got that right Sam??

I've booked the pump out station at the Marina just in case the flushing
cannot be done.

kind regards
Steven


On Fri, 9 Dec 2022 at 08:55, michael martin ***@***.***>
wrote:

> The other way to relieve pressure is the method we used
>
> Siphon off waste in the toilet bowl.
>
> You will know when the tank has been relieved of pressure when the bowl
> stops filling up with waste.
>
> Note a messy job but necessary when the system is completely blocked and
> you cant relieve pressure by having it open to the sea.
>
>
>
> cheers
>
>
>
> *From:* Steven Ring ***@***.***>
> *Sent:* Friday, 9 December 2022 5:49 AM
> *To:* Sam Delorenzo (Delorco) ***@***.***>
> *Cc:* ***@***.*** ***@***.***>;
> michael martin ***@***.***>; Peter Wain ***@***.***>;
> Peter Lucey ***@***.***>; Michelle Grybaitis <
> ***@***.***>; Bill Russo ***@***.***>; Peter Otteson <
> ***@***.***>; Margot & Bill Leslie <
> ***@***.***>; ***@***.***
> *Subject:* Re: Namadgi 3 WC black tank Management
>
>
>
> Firstly, thank you Sam for the brilliant diagrams and descriptions of our
> black water system. It is a complex system and we have three of them on
> board.
>
>
>
> Now to the immediate problem...
>
>
>
> The current problem with the toilet(s) on Namadgi relates to a blocked
> breather tube. A blocked breather tube leads to a diabolical situation
> known as a pressure lock
>
>
>
>    1. Namadgi has NO reliable waste water tank monitoring system,
>    consequently we regularly over-fill the black water tanks,  with a full
>    tank...
>    2. Continued pumping of waste into the black water tank forces all air
>    out of the system via the breather tube, which then fills with effluent
>    3. Effluent will spay out of the breather tube sometime attracting the
>    attention (and comments) of nearby swimmers or neighbours enjoying a BBQ
>    -- a brown stain will run down the topsides
>    4. This effluent dries at the end of the breather creating a solid
>    plug that will NOT allow air back into the system.
>    5. Water (and shit) are incompressible
>    6. Further pumping of effluent causes the plastic black water tank to
>    expand (and actually stretch) placing the vile contents under high pressure
>    7. Shit under pressure is a dangerous thing
>    8. Over a short time, this pressure will overcome the joker valve
>    force black water into the bowl -- this is the first visible sign of a
>    major problem -- *Stop using those heads!!*
>    9. The pressure of the black water at the top of the pump out tube is
>    likely to well exceed the air pressure at the deck
>    10. Consequently, removal of the deck pump-out cap will cause a geyser
>    of shit to erupt into the air!!  Seriously! *Don't be tempted to open
>    that cap.*
>
>
>
> So how to relieve this pressure lock?
>
>
>
>    1. Opening the black water sea cock may relieve some of the pressure,
>    but
>    2. It's not unusual for effluent to settle just above the sea cock and
>    for a blockage of it's own, however, more commonly,
>    3. *a little effluen*t will escape to sea, relieving the positive
>    pressure in the top part of the tank and causing the walls of the tank to
>    bow inwards.
>    4. However, the upper part of the system is still closed, with no air
>    vent to the atmosphere, so the black tank *will not empty to the sea*
>    despite the valve being open
>    5. Opening the pump out deck fitting at this stage is less dangerous
>    than before but is *a complete waste of time*
>    6. Sam's diagram shows the horrible reality, the pump out tube loops
>    from the deck fitting to the bottom of the tank...
>    7. Before air can get into the tank, the effluent in the tube (and
>    there is a lot of it) must enter the black water tank -- but there is no
>    room for that effluent
>    8. The situation is a perfect pressure lock from which there is no
>    easy exit
>
>
>
> Is there a solution? It depends where you are. At sea or away from a
> marina (where no pump out facility is available)
>
>    1. With black water seacock open...
>    2. Remove the cover to gain access to the black water tank
>    3. Detach the breather tube from the top of the tank (a Phillips head
>    screwdriver is needed to ease the jubilee clamp)
>    4. Don't be distressed by the inrush of air as the tanks suddenly
>    expands and the column of shit is dumped to the sea
>    5. at the earliest opportunity, use a pressure hose to clear the
>    breather line from the outside
>    6. reattach the breather tube once it has been cleared
>
>
>
> Good luck and please let us know how you get on.
>
>
>
> regards
>
> Steven
>
>
>
>
>
>
>
>
>
> On Thu, 8 Dec 2022 at 10:20, Sam Delorenzo (Delorco) <
> ***@***.***> wrote:
>
> Dear All
>
> There is certainly a need to ensure consistency on how to manage the Black
> Water system on Namadgi
>
> I have studied our system and experienced (and have cleared) the blockages
> (our foe).
>
> I offer the attached and below as a point of clarification and maybe
> assist understanding of a properly working system (our friend)
>
> I attach a series of sketches of how I understand the configuration on N3.
>
> The system provides many opportunities for failure, which include.
>
>    - Foreign object tank blockages (wet wipes) – Unfortunately, our tanks
>    will funnel these to outlets (see diagram 5)
>    - Calcification - see diagram 2 showing where wastewater sits under
>    normal circumstances and build up
>    - Breather line blockage - see diagrams 3 & 5 showing dual breather
>    overflow operation
>    - Sea cock blockage at the hull.
>    - Failure to verify that tanks have emptied at sea (the golden streak
>    is a beautiful sight)
>    - Incorrect/inadequate pumping sequence (there is only one way)
>    - Failed seals
>    - Black tanks over filled (can block the breather)
>    - Combination of any of the above
>
> Note: the best dye/tracing indicator is a litre of milk through the system
> for observation.
>
> Diagram 7 is a draft of how I think we can maintain a reliable system
> (when it is clear and unblocked).
>
> I would welcome any comments from experiences.
>
> I am preparing a presentation *Live on the boat* either during the
> working bee this month, or suitable date before South Pacific departure,
> and I will be asking all skippers and interested members to attend so we
> can all be constantly confident that the system is being maintained as best
> we can.
>
> It’s quite a simple process of preventative maintenance that must be
> understood and carried out by all Skippers.
>
>
>
>
>
> Regards,
>
> Sam DeLorenzo
>
> Mobile & WhatsApp: +61 411 165 835
>
> PO Box 3553, Manuka ACT  AUSTRALIA 2603
>
> ______________________________________________________________________________________
>
>
> IMPORTANT
> This e-mail and any accompanying attachments are intended for the sole use
> of the addressee(s) and may contain confidential information. If you are
> not the intended recipient (or authorised designee), please disregard the
> contents of this e-mail, notify the sender immediately and destroy all
> copies of this e-mail
>
>
>
> *From:* ***@***.*** ***@***.***>
> *Sent:* Thursday, 8 December 2022 09:09
> *To:* 'michael martin' ***@***.***>
> *Cc:* 'Peter Wain' ***@***.***>; 'Steven Ring' ***@***.***>;
> 'CORC Committee' ***@***.***>; 'Peter Lucey' <
> ***@***.***>; Sam Delorenzo (Delorco) <
> ***@***.***>; 'Paul & Sue Jones' <
> ***@***.***>
> *Subject:* RE: Defect Condition of Namadgi - 01Dec22 - Addendum #2
>
>
>
> Good morning Michael
>
>
>
> Agreed that something is awry.
>
>
>
> All holding tanks need to be easy to dump, and to have no back wash to the
> bowl.
>
>
>
> Is it possible to use a two litre container of vinegar flushed through
> with fresh water and simply left in the tank for a couple of days?  Perhaps
> Sam has a better approach?
>
>
>
> Kind regards
>
>
>
> Paul
>
>
>
> *From:* michael martin ***@***.***>
> *Sent:* Wednesday, 7 December 2022 9:37 PM
> *To:* ***@***.***
> *Cc:* Peter Wain ***@***.***>; Steven Ring ***@***.***>;
> CORC Committee ***@***.***>; Peter Lucey <
> ***@***.***>; Sam Delorenzo (Delorco) <
> ***@***.***>; Paul & Sue Jones ***@***.***>
> *Subject:* Re: Defect Condition of Namadgi - 01Dec22 - Addendum #2
>
>
>
> It is clear then that something is
>
> wrong with the toilet
>
> We bucketed out a lot of shitty water out of the toilet and the pumpout
> guy said it was completely full. The stuff that came out of the holding
> tank was heavy with waste and paper.
>
> Cheers
>
>
>
> Sent from my iPhone
>
>
>
> On 7 Dec 2022, at 12:22 pm, ***@***.*** wrote:
>
> ﻿
>
> Many thanks, Peter.
>
>
>
> That is not a feature of PredictWind with which I am familiar.  If it is
> in place (or can be set up on the new tablet), all good.
>
>
>
> If it’s not in the SOP, I may need to seek your assistance.
>
>
>
> Kind regards
>
>
>
> Paul
>
>
>
> *From:* Peter Wain ***@***.***>
> *Sent:* Wednesday, 7 December 2022 11:41 AM
> *To:* ***@***.*** ***@***.***>
> *Cc:* Steven Ring ***@***.***>; CORC Committee <
> ***@***.***>; Peter Lucey ***@***.***>; Sam
> Delorenzo (Delorco) ***@***.***>; Michael Martin <
> ***@***.***>; Paul & Sue Jones ***@***.***>
> *Subject:* Re: Defect Condition of Namadgi - 01Dec22 - Addendum #2
>
>
>
> Hi Paul, you mentioned Yellow Brick, but you may not be aware that our
> Predict Wind - Iridium Go already provides automatic tracking as well as
> the ability to send and receive e-mails, phone calls, SMS messages, and to
> receive weather forecasts, grib files, and weather routing.  We used it for
> all of those things in 2019. Regards, Peter.
>
> On Wed, 7 Dec 2022, 11:16 am ***@***.***, <
> ***@***.***> wrote:
>
> Good morning Steven
>
>
>
> Michael’s Skipper’s Report (Tuesday, 6 December 2022 9:34 PM) reminded me
> of another couple of problems.
>
>
>
>    1. It is unfortunate Michael leapt to conclusions “*overflowing toilet
>    onto the floor coming from the forward head which had overfilled because it
>    hadn’t been emptied*”.  I can advise that I emptied and flushed all
>    the tanks offshore on Thursday 01Dec22.
>    2. However, Peter and I had also noted a problem with all bowls, that
>    being after pumping the bowl dry, bowls back filled with liquid of various
>    shades.  I surmised it might be a syphoning issue as fluid drained back
>    down.  Peter thought perhaps it was the joker valve, which should prevent
>    the drain back.  Either way, whatever makes its way into the bowl presents
>    a serious problem in a seaway because the liquid slops onto the floor (as
>    Michael reported) and is then tramped through the yacht.  Definitely a
>    matter for urgent pre-deployment resolution.
>
>
>
> I apologise for having overlooked mentioning the heads problem.
>
>
>
>    1. The other item for consideration might be finally fitting a Yellow
>    Brick tracker.  This was considered for the Elan but deferred.  Perhaps its
>    time has come again with a South Pacific deployment?
>
>
>
> For consideration
>
>
>
> Regards
>
>
>
> Paul
>
>
>
> *From:* ***@***.*** ***@***.***>
> *Sent:* Tuesday, 6 December 2022 11:33 AM
> *To:* 'Steven Ring' ***@***.***>
> *Cc:* 'CORC Committee' ***@***.***>; 'Peter Lucey' <
> ***@***.***>; 'Sam Delorenzo (Delorco)' <
> ***@***.***>; ***@***.***' <
> ***@***.***>; ***@***.***' <
> ***@***.***>
> *Subject:* RE: Defect Condition of Namadgi - 01Dec22
>
>
>
> Good morning Steven
>
>
>
> Thank you.
>
>
>
> You mentioned elsewhere that my list might not be exhaustive.
>
>
>
> Can I offer the items below as a start point for further items for
> consideration:
>
>
>
>    1. Check sail repair kit (needles , palm etc)
>    2. Check range and appropriateness of sail repair patches
>    3. Check Defibrillator for condition and confirm battery is still in
>    date (or will not expire during the deployment)
>    4. Check Defibrillator kit still has scissors (for cutting away
>    clothing) and a razor (to clear area for pads)
>    5. Availability of air splints (both arm and leg)
>    6. Body bag (if someone dies half way across the Tasman, what is the
>    plan for the body?)
>    7. Emergency steering gear to be checked (and trialled?)
>    8. Locate and fit Perspex wash boards and hatch inserts
>    9. Shock cord and clip from inside the Lazarette hatch is missing.
>
>
>
> Dare I mention pillows and hospital grade pillow protectors?
>
>
>
> You advised that a rigging check, together with a keel and rudder
> inspection, have been completed.
>
>
>
> When I hear from the See-Blitz OEM, I’ll be in touch with you.
>
>
>
> Regards
>
>
>
> Paul
>
>
>
> *From:* Steven Ring ***@***.***>
> *Sent:* Sunday, 4 December 2022 10:55 PM
> *To:* ***@***.*** ***@***.***>
> *Cc:* CORC Committee ***@***.***>; Peter Lucey <
> ***@***.***>; Sam Delorenzo (Delorco) <
> ***@***.***>; ***@***.***;
> ***@***.***
> *Subject:* Re: Defect Condition of Namadgi - 01Dec22
>
>
>
> Thanks Paul,
>
>
>
> That’s quite a list. I’ll forward to the committee for discussion tomorrow
> night.
>
>
>
> Kind regards
>
> Steven
>
>
>
> On Sun, 4 Dec 2022 at 3:22 pm, ***@***.*** <
> ***@***.***> wrote:
>
> Good afternoon Steven
>
> This is the email I promised yesterday.
>
> I returned on Thurs evening (01Dec22) after 4 days on board Namadgi.  I
> have to say, that Namadgi’s condition is less than I expected and does not
> meet the Club’s stated safety standard.  To remind, The Canberra Ocean
> Racing Club (CORC) Safety Policy (7 May 2021) states “… the Club maintains
> the yacht and it’s [sic] equipment to a standard required for Category 1
> races as described in the Australian Sailing’s Special Regulations - Part 1
> for Racing Boats”.   In my view, Namadgi is not ready in all respects to
> undertake an ocean passage to NZ.
>
> *Potential Showstoppers*
>
>    1. The pin in the port pivot point of the platform is loose and was
>    found to be almost disengaged (the grub screw which locates the pin is
>    missing).  This represents a very high safety risk.
>    2. Oil in Sail drive, but it is below minimum mark on the dipstick.
>    This requires urgent attention.
>    3. No liferaft – I do not know when it is to be returned. Timing may
>    be very tight if Michael sails early in the week commencing Sun04Dec22.
>    4. Namadgi’s life jackets and PLBs not on board. I do not know when
>    they are to be returned. Timing may be very tight if Michael sails early in
>    the week commencing Sun04Dec22.
>    5. Both See-Blitz units are unserviceable.  After my discussion with
>    you, I removed both from Namadgi and have sent them to the OEM for
>    assessment.  The company is not optimistic from my description and CORC may
>    have to buy a new primary See-Blitz unit.  (Interestingly, the primary
>    See-Blitz was not set up correctly and would not have activated even if the
>    unit had been serviceable.)
>    6. HF radio is locked at a “DSC Watch” screen.  I was not able to
>    progress beyond this screen to select channels/frequencies.  Accordingly,
>    no test transmissions have been conducted. Neither you nor Sam Hughes was
>    able to offer a way ahead.  (The Club website link to the ICOM HF manual
>    takes you to the Iridium GO! Satellite Device Quick Start Guide.  Also, the
>    Club website suggests the HF radio is an ICOM M802.  The fitted unit is an
>    ICOM IC‑M801E).
>    7. The rear platform is manual only and cannot be secured in the
>    closed position (standing/sitting on it is dangerous) and in heavy seas
>    might allow ingress of seawater.
>    8. Drugs – I did not muster the drugs but understand that many
>    prescription Cat 1 drugs are out of date. (A list provided by Peter Lucey
>    is attached at the end of this email).
>    9. It appears there is a problem with forward water tanks.  The aft
>    tank lasted eight persons over 2 days.  The forward tanks were emptied in
>    quick succession even with no showers for the last 1.5 days.
>    10. The watermaker is unserviceable (query low-pressure water pump not
>    operational).
>    11. Garmin chart plotter screen locked up when zooming.
>    12. I have downloaded some PredictWind data to my iPad using the on
>    board Iridium Go.  However, I have not yet done so to the Tablet, which was
>    not on board.  It is not clear to me how the PredictWind suggested weather
>    routes are to be transferred from the tablet to the chart plotter.  A
>    method to transfer the PredictWind weather route to the chart plotter is
>    required.  This matter needs urgent resolution.
>    13. The marking on the anchor cable needs to be refreshed – there are
>    the remnants of two systems (paint and plastic inserts) which need
>    validating before deployment.  An end of chain warning would also be useful.
>    14. No internal rope grab rail fitted to allow safe movement through
>    Namadgi in heavy weather, after coming down the companionway.
>    15. Battery powered angle grinder not located.
>
>  *Other irritants*
>
>    1. There is movement in the grab rail of the port side of the
>    companionway.
>    2. Cockroaches were sighted on board.  Some were killed.  Baits were
>    laid before returning to Canberra.
>    3. Cockpit emergency knife is detaching.  Both sides of the Velcro
>    strip need to be removed, both surfaces cleaned (Acetone) and the Velcro
>    replaced (fluffy side on the yacht).
>    4. All lifelines are slack and need tightening.
>    5. The rear lifelines appear to interfere with the opening of the
>    Lazarette, where the fire extinguisher and other safety equipment is stored.
>    6. The lifelines and entry/egress point snap shackles need a thorough
>    check and distorted rings replaced.
>    7. The fire extinguisher in the Lazarette has detached from its
>    stowage and is not secured in the locker.  The screws in the extinguisher
>    bracket are dangerous.
>    8. Mainsail Track carriages – I located some track carriages but have
>    not confirmed that they are appropriate for the new sail, nor that the
>    breakdown (Full batten versus ordinary carriage) is appropriate.
>    9. Both dinghy oars are held together by tape.  One fitted oar blade
>    has part broken off and is split.
>    10. Emergency water exists but might benefit from a refresh.
>    11. Cabinet door two under Cutlery drawer doesn’t close properly –
>    needs adjustment.
>    12. At least two wire galley door restraints have failed such that the
>    door falls past horizontal - Port side aft and port side midships lower.
>    13. Starboard forward heads have a saltwater leak (onto the deck) when
>    pumping.
>    14. Toilet brushes were missing, and one toilet brush holder broken.
>    The purchased replacements are not suitable.
>    15. Boom bag is too tight – suggest a new front panel to add some 10cm
>    to the length of the bag.  Further, engaging and closing the zip on the new
>    sail is so difficult, I suspect many will not persevere.  Suggest the
>    sailmaker be invited to develop a more user-friendly solution.
>
> It is not clear to me from the Sailing Programme where time is available
> for a Maintenance Period to address these issues.
>
> For consideration
>
> Kind regards
>
> Paul
>
>  *Drugs on Board*
>
> *Expired drugs list attached (courtesy Peter Lucey)*
> Ibilex 500 20 Capsules) Audit Date: Expiry Date: 1/5/2023 Qty On Hand: 1
> Chlorsig 1% Eye Ointment 1g Audit Date: Expiry Date: 28/2/2023 Qty On
> Hand: 1
> Panamax Co (40 Tablets) Audit Date: Expiry Date: 31/7/2022 Qty On Hand: 1
> Prodeine Forte (20 Tabs) Audit Date: Expiry Date: 30/9/2023 Qty On Hand: 1
> Endone (20 Tablets) Audit Date: Expiry Date: 31/7/2022 Qty On Hand: 1
> Narcan (Naloxone Hydrochloride) 10x1ml ampoules Audit Date: Expiry Date:
> 30/4/2021 Qty On Hand: 1
> Morphine Hydrochloride (10mg for injection) 5x1ml ampoules Audit Date:
> Expiry Date: 28/2/2021 Qty On Hand: 1
> Aspen Adrenaline Injection (1mg/1ml) 5 ampoules Audit Date: Expiry Date:
> 30/4/2020 Qty On Hand: 1
> Maxolon 10 ampoules Audit Date: 2/5/2022 Expiry Date: 31/3/2020 Qty On
> Hand: 1
>
> 1ml Tuberculin Syringe Audit Date: Expiry Date: Qty On Hand: 1
> Terumo Needle (23G 32mm) Audit Date: Expiry Date: Qty On Hand: 5
> 10ml Syringe Audit Date: Expiry Date: Qty On Hand: 6
>
> --
>
> *Steven Ring*
>
> m: +61 417 495 268
>
>
>
>
> --
>
> *Steven Ring*
>
> m: +61 417 495 268
>


-- 
*Steven Ring*
m: +61 417 495 268

**Comment by smr547** [comment]
Thanks Steve
All looks good.
If possible, I would use fresh water wherever possible to flush if theboat is sitting for any time unused

Regards,
Sam DeLorenzo
Mobile & WhatsApp: +61 411 165 835
PO Box 3553, Manuka ACT  AUSTRALIA 2603
______________________________________________________________________________________
IMPORTANT
This e-mail and any accompanying attachments are intended for the sole use of the addressee(s) and may contain confidential information. If you are not the intended recipient (or authorised designee), please disregard the contents of this e-mail, notify the sender immediately and destroy all copies of this e-mail

From: Steven Ring ***@***.***>
Sent: Friday, 9 December 2022 12:49
To: michael martin ***@***.***>
Cc: Sam Delorenzo (Delorco) ***@***.***>; ***@***.*** ***@***.***>; Peter Wain ***@***.***>; Peter Lucey ***@***.***>; Michelle Grybaitis ***@***.***>; Bill Russo ***@***.***>; Peter Otteson ***@***.***>; Margot & Bill Leslie ***@***.***>; ***@***.***; ***@***.***
Subject: Re: Namadgi 3 WC black tank Management

Hi Michael,

That makes sense. The pressurised black water tank is forcing effluent back into the toilet bowl through the joker valve (which is a very inefficient one-way valve). Once sufficient ugly water has returned to the bowl, pressure will reduce in the black water tank and the flow will stop. Bucketing the effluent out is a great idea.... gone but not forgotten!!

Could you please flush fresh sea water into the deck pump out fitting -- two or three buckets please. This will clean out effluent from the pump out hose loop. All of these pipes are due to be pulled out over the next two weeks. Have I got that right Sam??

I've booked the pump out station at the Marina just in case the flushing cannot be done.

kind regards
Steven


On Fri, 9 Dec 2022 at 08:55, michael martin ***@***.******@***.***>> wrote:
The other way to relieve pressure is the method we used
Siphon off waste in the toilet bowl.
You will know when the tank has been relieved of pressure when the bowl stops filling up with waste.
Note a messy job but necessary when the system is completely blocked and you cant relieve pressure by having it open to the sea.

cheers

From: Steven Ring ***@***.******@***.***>>
Sent: Friday, 9 December 2022 5:49 AM
To: Sam Delorenzo (Delorco) ***@***.******@***.***>>
Cc: ***@***.******@***.***> ***@***.******@***.***>>; michael martin ***@***.******@***.***>>; Peter Wain ***@***.******@***.***>>; Peter Lucey ***@***.******@***.***>>; Michelle Grybaitis ***@***.******@***.***>>; Bill Russo ***@***.******@***.***>>; Peter Otteson ***@***.******@***.***>>; Margot & Bill Leslie ***@***.******@***.***>>; ***@***.******@***.***>
Subject: Re: Namadgi 3 WC black tank Management

Firstly, thank you Sam for the brilliant diagrams and descriptions of our black water system. It is a complex system and we have three of them on board.

Now to the immediate problem...

The current problem with the toilet(s) on Namadgi relates to a blocked breather tube. A blocked breather tube leads to a diabolical situation known as a pressure lock


  1.  Namadgi has NO reliable waste water tank monitoring system, consequently we regularly over-fill the black water tanks,  with a full tank...
  2.  Continued pumping of waste into the black water tank forces all air out of the system via the breather tube, which then fills with effluent
  3.  Effluent will spay out of the breather tube sometime attracting the attention (and comments) of nearby swimmers or neighbours enjoying a BBQ  -- a brown stain will run down the topsides
  4.  This effluent dries at the end of the breather creating a solid plug that will NOT allow air back into the system.
  5.  Water (and shit) are incompressible
  6.  Further pumping of effluent causes the plastic black water tank to expand (and actually stretch) placing the vile contents under high pressure
  7.  Shit under pressure is a dangerous thing
  8.  Over a short time, this pressure will overcome the joker valve force black water into the bowl -- this is the first visible sign of a major problem -- Stop using those heads!!
  9.  The pressure of the black water at the top of the pump out tube is likely to well exceed the air pressure at the deck
  10. Consequently, removal of the deck pump-out cap will cause a geyser of shit to erupt into the air!!  Seriously! Don't be tempted to open that cap.

So how to relieve this pressure lock?


  1.  Opening the black water sea cock may relieve some of the pressure, but
  2.  It's not unusual for effluent to settle just above the sea cock and for a blockage of it's own, however, more commonly,
  3.  a little effluent will escape to sea, relieving the positive pressure in the top part of the tank and causing the walls of the tank to bow inwards.
  4.  However, the upper part of the system is still closed, with no air vent to the atmosphere, so the black tank will not empty to the sea despite the valve being open
  5.  Opening the pump out deck fitting at this stage is less dangerous than before but is a complete waste of time
  6.  Sam's diagram shows the horrible reality, the pump out tube loops from the deck fitting to the bottom of the tank...
  7.  Before air can get into the tank, the effluent in the tube (and there is a lot of it) must enter the black water tank -- but there is no room for that effluent
  8.  The situation is a perfect pressure lock from which there is no easy exit

Is there a solution? It depends where you are. At sea or away from a marina (where no pump out facility is available)

  1.  With black water seacock open...
  2.  Remove the cover to gain access to the black water tank
  3.  Detach the breather tube from the top of the tank (a Phillips head screwdriver is needed to ease the jubilee clamp)
  4.  Don't be distressed by the inrush of air as the tanks suddenly expands and the column of shit is dumped to the sea
  5.  at the earliest opportunity, use a pressure hose to clear the breather line from the outside
  6.  reattach the breather tube once it has been cleared

Good luck and please let us know how you get on.

regards
Steven




On Thu, 8 Dec 2022 at 10:20, Sam Delorenzo (Delorco) ***@***.******@***.***>> wrote:
Dear All
There is certainly a need to ensure consistency on how to manage the Black Water system on Namadgi
I have studied our system and experienced (and have cleared) the blockages (our foe).
I offer the attached and below as a point of clarification and maybe assist understanding of a properly working system (our friend)
I attach a series of sketches of how I understand the configuration on N3.
The system provides many opportunities for failure, which include.

  *   Foreign object tank blockages (wet wipes) – Unfortunately, our tanks will funnel these to outlets (see diagram 5)
  *   Calcification - see diagram 2 showing where wastewater sits under normal circumstances and build up
  *   Breather line blockage - see diagrams 3 & 5 showing dual breather overflow operation
  *   Sea cock blockage at the hull.
  *   Failure to verify that tanks have emptied at sea (the golden streak is a beautiful sight)
  *   Incorrect/inadequate pumping sequence (there is only one way)
  *   Failed seals
  *   Black tanks over filled (can block the breather)
  *   Combination of any of the above
Note: the best dye/tracing indicator is a litre of milk through the system for observation.
Diagram 7 is a draft of how I think we can maintain a reliable system (when it is clear and unblocked).
I would welcome any comments from experiences.
I am preparing a presentation Live on the boat either during the working bee this month, or suitable date before South Pacific departure, and I will be asking all skippers and interested members to attend so we can all be constantly confident that the system is being maintained as best we can.
It’s quite a simple process of preventative maintenance that must be understood and carried out by all Skippers.


Regards,
Sam DeLorenzo
Mobile & WhatsApp: +61 411 165 835
PO Box 3553, Manuka ACT  AUSTRALIA 2603
______________________________________________________________________________________
IMPORTANT
This e-mail and any accompanying attachments are intended for the sole use of the addressee(s) and may contain confidential information. If you are not the intended recipient (or authorised designee), please disregard the contents of this e-mail, notify the sender immediately and destroy all copies of this e-mail

From: ***@***.******@***.***> ***@***.******@***.***>>
Sent: Thursday, 8 December 2022 09:09
To: 'michael martin' ***@***.******@***.***>>
Cc: 'Peter Wain' ***@***.******@***.***>>; 'Steven Ring' ***@***.******@***.***>>; 'CORC Committee' ***@***.******@***.***>>; 'Peter Lucey' ***@***.******@***.***>>; Sam Delorenzo (Delorco) ***@***.******@***.***>>; 'Paul & Sue Jones' ***@***.******@***.***>>
Subject: RE: Defect Condition of Namadgi - 01Dec22 - Addendum #2

Good morning Michael

Agreed that something is awry.

All holding tanks need to be easy to dump, and to have no back wash to the bowl.

Is it possible to use a two litre container of vinegar flushed through with fresh water and simply left in the tank for a couple of days?  Perhaps Sam has a better approach?

Kind regards

Paul

From: michael martin ***@***.******@***.***>>
Sent: Wednesday, 7 December 2022 9:37 PM
To: ***@***.******@***.***>
Cc: Peter Wain ***@***.******@***.***>>; Steven Ring ***@***.******@***.***>>; CORC Committee ***@***.******@***.***>>; Peter Lucey ***@***.******@***.***>>; Sam Delorenzo (Delorco) ***@***.******@***.***>>; Paul & Sue Jones ***@***.******@***.***>>
Subject: Re: Defect Condition of Namadgi - 01Dec22 - Addendum #2

It is clear then that something is
wrong with the toilet
We bucketed out a lot of shitty water out of the toilet and the pumpout guy said it was completely full. The stuff that came out of the holding tank was heavy with waste and paper.
Cheers

Sent from my iPhone

On 7 Dec 2022, at 12:22 pm, ***@***.******@***.***> wrote:
﻿
Many thanks, Peter.

That is not a feature of PredictWind with which I am familiar.  If it is in place (or can be set up on the new tablet), all good.

If it’s not in the SOP, I may need to seek your assistance.

Kind regards

Paul

From: Peter Wain ***@***.******@***.***>>
Sent: Wednesday, 7 December 2022 11:41 AM
To: ***@***.******@***.***> ***@***.******@***.***>>
Cc: Steven Ring ***@***.******@***.***>>; CORC Committee ***@***.******@***.***>>; Peter Lucey ***@***.******@***.***>>; Sam Delorenzo (Delorco) ***@***.******@***.***>>; Michael Martin ***@***.******@***.***>>; Paul & Sue Jones ***@***.******@***.***>>
Subject: Re: Defect Condition of Namadgi - 01Dec22 - Addendum #2

Hi Paul, you mentioned Yellow Brick, but you may not be aware that our Predict Wind - Iridium Go already provides automatic tracking as well as the ability to send and receive e-mails, phone calls, SMS messages, and to receive weather forecasts, grib files, and weather routing.  We used it for all of those things in 2019. Regards, Peter.
On Wed, 7 Dec 2022, 11:16 am ***@***.******@***.***>, ***@***.******@***.***>> wrote:
Good morning Steven

Michael’s Skipper’s Report (Tuesday, 6 December 2022 9:34 PM) reminded me of another couple of problems.


  1.  It is unfortunate Michael leapt to conclusions “overflowing toilet onto the floor coming from the forward head which had overfilled because it hadn’t been emptied”.  I can advise that I emptied and flushed all the tanks offshore on Thursday 01Dec22.
  2.  However, Peter and I had also noted a problem with all bowls, that being after pumping the bowl dry, bowls back filled with liquid of various shades.  I surmised it might be a syphoning issue as fluid drained back down.  Peter thought perhaps it was the joker valve, which should prevent the drain back.  Either way, whatever makes its way into the bowl presents a serious problem in a seaway because the liquid slops onto the floor (as Michael reported) and is then tramped through the yacht.  Definitely a matter for urgent pre-deployment resolution.

I apologise for having overlooked mentioning the heads problem.


  1.  The other item for consideration might be finally fitting a Yellow Brick tracker.  This was considered for the Elan but deferred.  Perhaps its time has come again with a South Pacific deployment?

For consideration

Regards

Paul

From: ***@***.******@***.***> ***@***.******@***.***>>
Sent: Tuesday, 6 December 2022 11:33 AM
To: 'Steven Ring' ***@***.******@***.***>>
Cc: 'CORC Committee' ***@***.******@***.***>>; 'Peter Lucey' ***@***.******@***.***>>; 'Sam Delorenzo (Delorco)' ***@***.******@***.***>>; ***@***.******@***.***>' ***@***.******@***.***>>; ***@***.******@***.***>' ***@***.******@***.***>>
Subject: RE: Defect Condition of Namadgi - 01Dec22

Good morning Steven

Thank you.

You mentioned elsewhere that my list might not be exhaustive.

Can I offer the items below as a start point for further items for consideration:


  1.  Check sail repair kit (needles , palm etc)
  2.  Check range and appropriateness of sail repair patches
  3.  Check Defibrillator for condition and confirm battery is still in date (or will not expire during the deployment)
  4.  Check Defibrillator kit still has scissors (for cutting away clothing) and a razor (to clear area for pads)
  5.  Availability of air splints (both arm and leg)
  6.  Body bag (if someone dies half way across the Tasman, what is the plan for the body?)
  7.  Emergency steering gear to be checked (and trialled?)
  8.  Locate and fit Perspex wash boards and hatch inserts
  9.  Shock cord and clip from inside the Lazarette hatch is missing.

Dare I mention pillows and hospital grade pillow protectors?

You advised that a rigging check, together with a keel and rudder inspection, have been completed.

When I hear from the See-Blitz OEM, I’ll be in touch with you.

Regards

Paul

From: Steven Ring ***@***.******@***.***>>
Sent: Sunday, 4 December 2022 10:55 PM
To: ***@***.******@***.***> ***@***.******@***.***>>
Cc: CORC Committee ***@***.******@***.***>>; Peter Lucey ***@***.******@***.***>>; Sam Delorenzo (Delorco) ***@***.******@***.***>>; ***@***.******@***.***>; ***@***.******@***.***>
Subject: Re: Defect Condition of Namadgi - 01Dec22

Thanks Paul,

That’s quite a list. I’ll forward to the committee for discussion tomorrow night.

Kind regards
Steven

On Sun, 4 Dec 2022 at 3:22 pm, ***@***.******@***.***> ***@***.******@***.***>> wrote:
Good afternoon Steven
This is the email I promised yesterday.
I returned on Thurs evening (01Dec22) after 4 days on board Namadgi.  I have to say, that Namadgi’s condition is less than I expected and does not meet the Club’s stated safety standard.  To remind, The Canberra Ocean Racing Club (CORC) Safety Policy (7 May 2021) states “… the Club maintains the yacht and it’s [sic] equipment to a standard required for Category 1 races as described in the Australian Sailing’s Special Regulations - Part 1 for Racing Boats”.   In my view, Namadgi is not ready in all respects to undertake an ocean passage to NZ.
Potential Showstoppers

  1.  The pin in the port pivot point of the platform is loose and was found to be almost disengaged (the grub screw which locates the pin is missing).  This represents a very high safety risk.
  2.  Oil in Sail drive, but it is below minimum mark on the dipstick.  This requires urgent attention.
  3.  No liferaft – I do not know when it is to be returned. Timing may be very tight if Michael sails early in the week commencing Sun04Dec22.
  4.  Namadgi’s life jackets and PLBs not on board. I do not know when they are to be returned. Timing may be very tight if Michael sails early in the week commencing Sun04Dec22.
  5.  Both See-Blitz units are unserviceable.  After my discussion with you, I removed both from Namadgi and have sent them to the OEM for assessment.  The company is not optimistic from my description and CORC may have to buy a new primary See-Blitz unit.  (Interestingly, the primary See-Blitz was not set up correctly and would not have activated even if the unit had been serviceable.)
  6.  HF radio is locked at a “DSC Watch” screen.  I was not able to progress beyond this screen to select channels/frequencies.  Accordingly, no test transmissions have been conducted. Neither you nor Sam Hughes was able to offer a way ahead.  (The Club website link to the ICOM HF manual takes you to the Iridium GO! Satellite Device Quick Start Guide.  Also, the Club website suggests the HF radio is an ICOM M802.  The fitted unit is an ICOM IC‑M801E).
  7.  The rear platform is manual only and cannot be secured in the closed position (standing/sitting on it is dangerous) and in heavy seas might allow ingress of seawater.
  8.  Drugs – I did not muster the drugs but understand that many prescription Cat 1 drugs are out of date. (A list provided by Peter Lucey is attached at the end of this email).
  9.  It appears there is a problem with forward water tanks.  The aft tank lasted eight persons over 2 days.  The forward tanks were emptied in quick succession even with no showers for the last 1.5 days.
  10. The watermaker is unserviceable (query low-pressure water pump not operational).
  11. Garmin chart plotter screen locked up when zooming.
  12. I have downloaded some PredictWind data to my iPad using the on board Iridium Go.  However, I have not yet done so to the Tablet, which was not on board.  It is not clear to me how the PredictWind suggested weather routes are to be transferred from the tablet to the chart plotter.  A method to transfer the PredictWind weather route to the chart plotter is required.  This matter needs urgent resolution.
  13. The marking on the anchor cable needs to be refreshed – there are the remnants of two systems (paint and plastic inserts) which need validating before deployment.  An end of chain warning would also be useful.
  14. No internal rope grab rail fitted to allow safe movement through Namadgi in heavy weather, after coming down the companionway.
  15. Battery powered angle grinder not located.
 Other irritants

  1.  There is movement in the grab rail of the port side of the companionway.
  2.  Cockroaches were sighted on board.  Some were killed.  Baits were laid before returning to Canberra.
  3.  Cockpit emergency knife is detaching.  Both sides of the Velcro strip need to be removed, both surfaces cleaned (Acetone) and the Velcro replaced (fluffy side on the yacht).
  4.  All lifelines are slack and need tightening.
  5.  The rear lifelines appear to interfere with the opening of the Lazarette, where the fire extinguisher and other safety equipment is stored.
  6.  The lifelines and entry/egress point snap shackles need a thorough check and distorted rings replaced.
  7.  The fire extinguisher in the Lazarette has detached from its stowage and is not secured in the locker.  The screws in the extinguisher bracket are dangerous.
  8.  Mainsail Track carriages – I located some track carriages but have not confirmed that they are appropriate for the new sail, nor that the breakdown (Full batten versus ordinary carriage) is appropriate.
  9.  Both dinghy oars are held together by tape.  One fitted oar blade has part broken off and is split.
  10. Emergency water exists but might benefit from a refresh.
  11. Cabinet door two under Cutlery drawer doesn’t close properly – needs adjustment.
  12. At least two wire galley door restraints have failed such that the door falls past horizontal - Port side aft and port side midships lower.
  13. Starboard forward heads have a saltwater leak (onto the deck) when pumping.
  14. Toilet brushes were missing, and one toilet brush holder broken.  The purchased replacements are not suitable.
  15. Boom bag is too tight – suggest a new front panel to add some 10cm to the length of the bag.  Further, engaging and closing the zip on the new sail is so difficult, I suspect many will not persevere.  Suggest the sailmaker be invited to develop a more user-friendly solution.
It is not clear to me from the Sailing Programme where time is available for a Maintenance Period to address these issues.
For consideration
Kind regards
Paul
 Drugs on Board
Expired drugs list attached (courtesy Peter Lucey)
Ibilex 500 20 Capsules) Audit Date: Expiry Date: 1/5/2023 Qty On Hand: 1
Chlorsig 1% Eye Ointment 1g Audit Date: Expiry Date: 28/2/2023 Qty On Hand: 1
Panamax Co (40 Tablets) Audit Date: Expiry Date: 31/7/2022 Qty On Hand: 1
Prodeine Forte (20 Tabs) Audit Date: Expiry Date: 30/9/2023 Qty On Hand: 1
Endone (20 Tablets) Audit Date: Expiry Date: 31/7/2022 Qty On Hand: 1
Narcan (Naloxone Hydrochloride) 10x1ml ampoules Audit Date: Expiry Date: 30/4/2021 Qty On Hand: 1
Morphine Hydrochloride (10mg for injection) 5x1ml ampoules Audit Date: Expiry Date: 28/2/2021 Qty On Hand: 1
Aspen Adrenaline Injection (1mg/1ml) 5 ampoules Audit Date: Expiry Date: 30/4/2020 Qty On Hand: 1
Maxolon 10 ampoules Audit Date: 2/5/2022 Expiry Date: 31/3/2020 Qty On Hand: 1
1ml Tuberculin Syringe Audit Date: Expiry Date: Qty On Hand: 1
Terumo Needle (23G 32mm) Audit Date: Expiry Date: Qty On Hand: 5
10ml Syringe Audit Date: Expiry Date: Qty On Hand: 6
--
Steven Ring
m: +61 417 495 268


--
Steven Ring
m: +61 417 495 268


--
Steven Ring
m: +61 417 495 268

**Comment by smr547** [comment]
> We have cleaned the forward heads
> 
> @Bill-Russo will be interested

This is come text

**Comment by smr547** [comment]
this issue should be addressed by task #635 planned for the working bee. Closing this issue with a review to reopening it if the not fixe at the working bee.
